test.sh

```diff
@@ -18,12 +18,10 @@ cd tests
 
 LUA=../src/lua
 
-# Non-serialised compilation tests
-# YKFIXME: The following tests are known to work with non-serialised JIT
-for test in api bwcoercion closure code events \
-    gengc pm tpack tracegc vararg goto cstack locals; do
-    YKD_SERIALISE_COMPILATION=0 ${LUA} -e"_U=true" ${test}.lua
+for serialise in 0 1; do
+    for test in api bwcoercion closure code events \
+                gengc pm tpack tracegc vararg goto cstack locals; do
+        echo "### YKD_SERIALISE_COMPILATION=$serialise $test.lua ###"
```

Comment:
I'm pretty sure buildbot echoes the lines it executes so the `echo` is unnecessary, but please check as I might be wrong!

Reply:
Ah, I just reverted it to the way it was [before](https://github.com/ykjit/yklua/commit/4bcdc9b28c9c77aec99b177d28f54a739af79430#diff-0693c60a7dfecde2f66e3572d6a12d454a8382abf4ab33653e4814929dfb6b94R50) my test changes

But will test it in Buildbot

Reply:
Doesn't look like buildbot echos the lines it executes 👉 https://ci.soft-dev.org/#/builders/1/builds/2520/steps/2/logs/stdio

Reply:
Oh, I see, that's because thisn't `.buildbot.sh`. I see.